command/plan.go

```diff
@@ -48,7 +49,15 @@ General Options:
 
   -force-count
     Use the taskgroup count from the Nomad jobfile instead of the count that
-    is currently set in a running job.
+	is currently set in a running job.
+
+  -output-format
```

Comment:
~`-output-format` is perhaps redundant? maybe just `-format`?~

update: i see that this is drawing a distinction against `-log-format`... 🤔